drone/exec/exec.go

```diff
@@ -327,6 +328,15 @@ func exec(c *cli.Context) error {
 			workspacePath = c.String("workspace-path")
 		}
 		dir, _ := filepath.Abs(filepath.Dir(file))
+
+		if runtime.GOOS == "windows" {
+			base := filepath.VolumeName(dir)
+			if len(base) == 2 {
+				dir = dir[2:]
```

Comment:
maybe instead of `dir = dir[2:]` we could do `dir = strings.TrimPrefix(dir, base)`? The only reason I suggest this is that performing slice operations without checking bounds sometimes makes me nervous, since I don't know all the possible values we could encounter